### PR TITLE
fix(Splitbutton) accessibility - screen reader fixes

### LINKED ISSUE #2083: SplitButton - screen readers - cursor/browse mode and disabled splitButton
Go through the points and fix it or report it as JAWS bugs. Please test it as well with NVDA and VoiceOver.
14.	[Screen readers]: In cursor mode, when navigating to “New conversation” disabled button JAWS reads as “New conversation button menu  Unavailable  menu”. (Expected: Screen reader should read as “New conversation split button menu Unavailable)
15.	[Screen readers]: On “More options” disabled button Screen reader reads as “Unlabeled 1 button menu Unavailable menu” (this will not repro when point 4 is fixed)

### PR DESCRIPTION
**Pull request by kolaps33**
fixes
#2083
#2080

 removing role='none' from menuButtonBehavior as this was not really needed.

### CODE REVIEW

File: packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts [hunk 1]
```diff
@@ -25,6 +26,8 @@ const splitButtonBehavior: Accessibility = props => {
             keyCombinations: [
               { keyCode: keyboardKey.Escape },
               { keyCode: keyboardKey.ArrowUp, altKey: true },
+              { keyCode: keyboardKey.Tab, shiftKey: false },
```
Comment (mnajdova) [inline packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts:39]: Are we absolutely sure that the items inside the menu cannot have elements that can be focused?

File: packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import menuButtonBehavior from '../MenuButton/menuButtonBehavior'
 const splitButtonBehavior: Accessibility = props => {
   const splitButtonMenuButtonBehavior = props => {
     const menuButtonBehaviorData = menuButtonBehavior(props)
+    menuButtonBehaviorData.attributes.trigger['aria-haspopup'] = undefined
```
Comment (mnajdova) [inline packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts:25]: Shouldn't we add more info in the @specification?

File: packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts (hunk 1 above)
Comment (jurokapsiar) [reply packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts:39]: yes, menu can only contain menu items, nothing else.

File: packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts (hunk 2 above)
Comment (kolaps33) [reply packages/accessibility/src/behaviors/SplitButton/splitButtonBehavior.ts:25]: as aria-popup shouldn't be add I am not writing to the description, that we are not adding it, but add the test to verify it :)

**Review by jurokapsiar — APPROVED** [review]

**Pull request merged**